packages/swayze/lib/src/widgets/table_body/selections/primary_selection/primary_selection.dart

```diff
@@ -91,9 +98,13 @@ class _PrimarySelectionState extends State<PrimarySelection>
         color: selectionStyle.backgroundColor,
         border: getVisibleBorder(range, borderSide).toFlutterBorder(),
       ),
+      handleStyle: handleStyle,
       duration: styleContext.selectionAnimationDuration,
       activeCellRect: widget.activeCellRect,
       isSingleCell: isSingleCell,
+      allowsHandle: selectionModel is CellUserSelectionModel
+          ? selectionModel.type != CellUserSelectionType.fill
+          : true,
```

Comment:
Isn't this widget always building the primary selection? The primary selection should never be the fill selection, right? So shouldnt this always be true?

Reply:
This has been removed as we have a FillSelectionModel now.